bagel/tests/test_utility.py

```diff
@@ -60,29 +62,68 @@ def test_all_used_namespaces_have_urls(
         ), f"The namespace '{prefix}' was used in the data dictionary, but was not defined in the @context."
 
 
-def test_schema_invalid_column_raises_error():
-    partial_data_dict = (
-        {
-            "participant_id": {
-                "Description": "A participant ID",
-                "Annotations": {
-                    "IsAbout": {
-                        "TermURL": "nb:ParticipantID",
-                        "Label": "Unique participant identifier",
-                    }
+@pytest.mark.parametrize(
+    "partial_data_dict, invalid_item",
```

Comment:
can we rename `invalid_item` to something more descriptive? Maybe `invalid_column_name`?